### PR TITLE
update client auth k8s faq

### PR DESCRIPTION
**Pull request by cswatt**
### What does this PR do?
1. Documents the kubeconfig approach for API server validation.
2. Updates existing FAQ to use Agent 6 env vars.
3. Corrects existing FAQ's grammar/style.

### Motivation
Support request.

### Preview link

https://docs-staging.datadoghq.com/cswatt/kubeconfig-approach/integrations/faq/client-authentication-against-the-apiserver-and-kubelet/

### CODE REVIEW

File: content/en/integrations/faq/client-authentication-against-the-apiserver-and-kubelet.md [hunk 1]
```diff
@@ -28,5 +28,16 @@ The default for kubelet traffic is to first try to use the read-only port that d
 kubelet_tls_verify: True
 ```
 
+## Alternate Option: kubeconfig
+
+Alternately, you can also use [kubeconfig][3] for API server authentication. Use the `DD_KUBERNETES_KUBECONFIG_PATH` environment variable to specify the path, or the equivalent option in [datadog.yaml][4]
```
Comment (ruthnaebeck) [inline content/en/integrations/faq/client-authentication-against-the-apiserver-and-kubelet.md]:
```suggestion
Alternately, you can use [kubeconfig][3] for API server authentication. Use the `DD_KUBERNETES_KUBECONFIG_PATH` environment variable to specify the path, or the equivalent option in [datadog.yaml][4]
```

File: content/en/integrations/faq/client-authentication-against-the-apiserver-and-kubelet.md [hunk 2]
```diff
@@ -13,13 +13,13 @@ kubelet_client_crt: /path/to/client.crt
 kubelet_client_key: /path/to/client.key
 ```
 
-## Server Authentication for apiserver and kubelet
+## Server Authentication for API server and kubelet
```
Comment (ruthnaebeck) [inline content/en/integrations/faq/client-authentication-against-the-apiserver-and-kubelet.md]:
```suggestion
## Server authentication for API server and kubelet
```

File: content/en/integrations/faq/client-authentication-against-the-apiserver-and-kubelet.md [hunk 3]
```diff
@@ -28,5 +28,16 @@ The default for kubelet traffic is to first try to use the read-only port that d
 kubelet_tls_verify: True
 ```
 
+## Alternate Option: kubeconfig
```
Comment (ruthnaebeck) [inline content/en/integrations/faq/client-authentication-against-the-apiserver-and-kubelet.md]:
```suggestion
## Alternate option: kubeconfig
```

File: content/en/integrations/faq/client-authentication-against-the-apiserver-and-kubelet.md [hunk 4]
```diff
@@ -28,5 +28,16 @@ The default for kubelet traffic is to first try to use the read-only port that d
...
+
+```
+kubernetes_kubeconfig_path: /path/to/file
+```
+
+
 [1]: https://kubernetes.io/docs/concepts/configuration/secret/#using-secrets-as-files-from-a-pod
 [2]: https://kubernetes.io/docs/concepts/configuration/secret
+[3]: https://kubernetes.io/docs/concepts/configuration/organize-cluster-access-kubeconfig
+[4]: https://github.com/DataDog/datadog-agent/blob/a9e0c4f534482170817300edb3cce01df9abea4a/pkg/config/config_template.yaml#L687-L692
```
Comment (ruthnaebeck) [inline content/en/integrations/faq/client-authentication-against-the-apiserver-and-kubelet.md]: Should we link to a static version of `datadog.yaml`?

Comment (cswatt) [reply content/en/integrations/faq/client-authentication-against-the-apiserver-and-kubelet.md]: i think config_template is best actually

**Pull request merged**